src/jukebox/jukebox/plugs.py

```diff
@@ -0,0 +1,660 @@
+                raise e
+
+
+def load_all_unnamed(packages_unnamed: Iterable[str], prefix: Optional[str] = None, ignore_errors=False):
+    """Load all packages in packages_unnamed with default names"""
+    for package in packages_unnamed:
+        try:
+            load(package, prefix=prefix)
+        except Exception as e:
+            if ignore_errors:
+                logger.warning(f"Ignoring failed package load for '{package}'")
+            else:
+                raise e
+
+
+def load_all_finalize():
+    """Calls all functions registered with @finalize from all loaded modules in the order they were loaded
+
+    This must be executed after the last plugin package is loaded"""
+    # Preserve loading-order for finalize order:
+    # Order of modules in dictionary is preserved in Python >= 3.7.
+    #  ... Else use OrdredDict for _PACKAGES??
+    # https://realpython.com/python-ordereddict/
+    assert sys.version_info.major >= 3 and sys.version_info.minor >= 7
+    for loaded_as, pack in _PLUGINS.items():
+        for func in pack.finalizer:
+            logger.debug(f"Package load finalizer: calling {loaded_as}.{func.__name__}()")
+            func()
+
+
+def call(package: str, plugin: str, method: Optional[str] = None, *,
+         args=(), kwargs=None, as_thread: bool = False) -> Any:
+    """
+    Call a function/method from the loaded plugins
+
+    - package.plugin(*args, **kwargs)
+        where plugin is a function or a callable instance of a class
+    - package.plugin.method(*args, **kwargs)
+        where plugin is a class instance
+        and method must have the attribute 'plugin_callable' = True
+
+    Developers notes:
+        There is no logger in this function as they all belong up-level where the exceptions are handled
+        If you want logger messages instead of exceptions, use call_ignore_errors
+
+    :param package: Name of the plugin package in which to look for function/class instance
+    :param plugin: Function name or instance name of a class
+    :param method: Method name when accessing a class instance' method. Leave at None if unneeded.
+    :param as_thread: Run the callable in separate daemon thread.
+    There is no return value from the callable in this case! The return value is the thread object.
+    Also note that Exceptions in the Thread must be handled in the Thread and are not propagated to the main Thread.
+    All threads are started as daemon threads with terminate upon main program termination.
+    There is not stop-thread mechanism. This is intended for short lived threads.
+    :param args: Arguments passed to callable
+    :param kwargs: Keyword arguments passed to callable
+    :return: The return value from the called function, or, if started as thread the thread object
+    """
+    if logger_call.isEnabledFor(logging.DEBUG):
+        m = f".{method}" if method is not None else ''
+        logger_call.debug(f"Calling: {package}.{plugin}{m}(args={args}, kwargs={kwargs})")
```

Comment:
I was able to run the PR in the Docker environment successfully. this command though was called once a second. Is this intentional?

```
jukebox    | 20.07.2021 12:29:05 -  572:plugs.py             - jb.plugin.call - Thread-238 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:06 -  572:plugs.py             - jb.plugin.call - Thread-239 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:06 -  572:plugs.py             - jb.plugin.call - Thread-240 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:06 -  572:plugs.py             - jb.plugin.call - Thread-241 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:06 -  572:plugs.py             - jb.plugin.call - Thread-242 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:07 -  572:plugs.py             - jb.plugin.call - Thread-243 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:07 -  572:plugs.py             - jb.plugin.call - Thread-244 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:07 -  572:plugs.py             - jb.plugin.call - Thread-245 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:07 -  572:plugs.py             - jb.plugin.call - Thread-246 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:08 -  572:plugs.py             - jb.plugin.call - Thread-247 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:08 -  572:plugs.py             - jb.plugin.call - Thread-248 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:08 -  572:plugs.py             - jb.plugin.call - Thread-249 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:08 -  572:plugs.py             - jb.plugin.call - Thread-250 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:09 -  572:plugs.py             - jb.plugin.call - Thread-251 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:09 -  572:plugs.py             - jb.plugin.call - Thread-252 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:09 -  572:plugs.py             - jb.plugin.call - Thread-253 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:09 -  572:plugs.py             - jb.plugin.call - Thread-254 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:10 -  572:plugs.py             - jb.plugin.call - Thread-255 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
jukebox    | 20.07.2021 12:29:10 -  572:plugs.py             - jb.plugin.call - Thread-256 - DEBUG    - Calling: volume.ctrl.get_volume(args=(), kwargs=None)
```

Reply:
Expected, as the playermpd checks the volume though ALSA as part of the status update. Not very use, I agree. Changed.